app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/model/message/LocationMessage.java

```diff
@@ -6,18 +6,22 @@
 
```

Comment:
You can add import static org.junit.Assert.assertEquals;
Then you only need to call assertEquals() not Asset.assertEquals